packages/manager/modules/pci/src/projects/project/project.html

```diff
@@ -1,9 +1,6 @@
 <div class="container-fluid h-100">
     <div class="row h-100">
-        <div class="col-md-2 p-0 h-100 d-md-flex flex-column"
-            data-ng-class="{
-                'd-none': !$ctrl.isSidebarOpen
-            }">
+        <div class="col-md-2 p-0 h-100 d-none d-lg-flex flex-column">
```

Comment:
Classes could be sorted to at least having a better understanding about what's going on, nope?

Such as defining width and margins, then tell how it could be displayed based on breakpoints for instance.

```suggestion
        <div class="h-100 p-0 d-none col-md-2 d-lg-flex flex-column">
```

Is that make sense?